src/AnnotationServiceProvider.php

```diff
@@ -36,14 +36,28 @@ public function boot(Application $app)
         $annotationService = $app['annot'];
 
         // Process annotations for all controllers in given directory
-        if ($app->offsetExists('annot.controllerDir') && strlen($app['annot.controllerDir']) > 0) {
+        if($app->offsetExists('annot.controllerDir') && is_array($app['annot.controllerDir']) && count($app['annot.controllerDir'])) {
```

Comment:
Please add a space before the parenthesis here and on lines 42 and 47.  Also, clean up extra whitespace on lines 57 and 60.